src/io/sampling.jl

```diff
@@ -179,90 +129,70 @@ Notes:
 
 # Arguments
 - `dom` : Domain
-- `n` : Int
-- `component` : Type, e.g. CriteriaWeights
-- `sampler` : type of sampler to use.
+- `n` : Number of samples to create
+- `component` : Component type, e.g. CriteriaWeights
+- `sampler` : Sampler to use
 
 # Returns
 Scenario specification
 """
-function sample(dom::Domain, n::Int, component::Type, sampler=SobolSample())::DataFrame
+function sample(dom::Domain, n::Int, component::Type, sampler=SobolSample(R=OwenScramble(base=2, pad=32)))::DataFrame
     n > 0 ? n : throw(DomainError(n, "`n` must be > 0"))
 
     spec = component_params(dom.model, component)
     return sample(spec, n, sampler)
 end
 
 """
-    sample(spec::DataFrame, n::Int, sampler=SobolSample())::DataFrame
+    sample(spec::DataFrame, n::Int, sample_method=SobolSample(R=OwenScramble(base=2, pad=32)))::DataFrame
 
 Create samples and rescale to distribution defined in the model spec.
 
 # Arguments
 - `spec` : DataFrame containing model parameter specifications.
 - `n` : number of samples to generate.
-- `sampler` : type of sampler to use.
+- `sample_method` : type of sampler to use.
```

Comment:
Ugh, oversight on my part.

`sampler` should not be an argument because it's a method name, so I switched to `sample_method` to avoid confusion. I'll fix in the morning.